chart/values.yaml

```diff
@@ -42,6 +42,59 @@ tolerations: []
 # Add common labels to all objects and pods defined in this chart.
 labels: {}
 
+# Ingress configuration
+ingress:
+  # Enable ingress resource
+  enabled: false
```

Comment:
Just thinking should we move this `enabled` conditional down to each resource? 
So `ingress.web.enabled` and `ingress.flower.enabled`
I may want `web` enabled, but `flower` ingress disabled if I have `Celery` as executor.

Reply:
@aksakalli WDYT about this suggestion?

Reply:
There is no option to disable `flower` if you use `CeleryExecutor` (unlike the [stable/helm](https://github.com/helm/charts/blob/8736171868dd9ba8e73584f3fc486ee7e341718b/stable/airflow/templates/ingress-flower.yaml#L1)). I don't think it is common that someone would like to deploy it with an ingress record for `webserver` but not for `flower`.